src/test/java/com/exasol/adapter/document/documentnode/parquet/RowRecordNodeTest.java

```diff
@@ -0,0 +1,40 @@
+package com.exasol.adapter.document.documentnode.parquet;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.adapter.document.documentnode.DocumentStringValue;
+import com.exasol.parquetio.data.Row;
+
+class RowRecordNodeTest {
+
```

Comment:
I would suggest here to create row instance instead of mocking, since it comes as dependency.